### ISSUE TITLE
event slot dont return classes

### ISSUE THREAD

**Issue by windows1087** [richardtallent/vue-simple-calendar#50]
I have the version `2.1.3` of this package installed. I try to change the event markup a bit:

```
 <div slot="event" slot-scope="e"
             class="event"
             :class="e.classes"
             :key="e.id">
          {{ e }}
        </div>
```

If i output {{ e }} it returns me 

```
 { "event": { "startDate": "2018-03-13", "id": 677, "title": "My Series" }, "weekStartDate": "2018-03-10T23:00:00.000Z" } 
```

I also tried it with `slot-scope="props"` which you say here https://github.com/richardtallent/vue-simple-calendar/issues/44 but there is an error in your example code (e does not exists), and it does also not work.

**Comment by richardtallent** [comment]
This is a known issue in 2.1.3. I've accepted a PR in #42 to fix this, which you can see now in the master branch here. That change will be in 2.2.0, which I'm finishing up now (webpack 4 migration headaches).

With this change, `event` will contain the "normalized" event (with classes, end date, etc.), and `event.originalEvent` will contain the original event you pass in.

I'll leave this open and close it with 2.2.0.

**Comment by richardtallent** [comment]
This should now be fixed in 2.2.0, just released!
